nodejs12.x/cookiecutter-aws-sam-hello-nodejs/README.md

```diff
@@ -10,8 +10,7 @@ A cookiecutter template to create a NodeJS Hello world boilerplate using [Server
 
 Generate a boilerplate template in your current project directory using the following syntax:
 
-* **NodeJS 10**: `sam init --runtime nodejs10.x`
-* **NodeJS 8**: `sam init --runtime nodejs8.10`
+* **NodeJS 12**: `sam init --runtime nodejs12.x`
```

Comment:
Node 10 is still supported. Shouldn't we still include it here since customers may not want to update their application's code/syntax?

Reply:
These are Language specific templates. If they are under the Node 10 folder they get the Node 10 option